generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js

```diff
@@ -6,8 +6,8 @@ import PropTypes from 'prop-types';
 
 import Routes from '../../../../constants/routes';
 
-const DEFAULT_PUBLIC_ROUTE = Routes.LOGIN;
-const DEFAULT_PRIVATE_ROUTE = Routes.HOME;
+const DEFAULT_PUBLIC_ROUTE = Routes.HOME;
```

Comment:
Is this ok? Home can not be the default public route. You need to be logged in order to access private routes. Check this.